libhb/hb.h

```diff
@@ -48,6 +49,8 @@ int           hb_check_update( hb_handle_t * h, char ** version );
 char *        hb_dvd_name( char * path );
 void          hb_dvd_set_dvdnav( int enable );
 
+int           hb_get_opencl_enabled (hb_handle_t * h);
```

Comment:
Wasn't aware we had a preference either way. It seems ot lean more towards space both sides.
I'll change.